Update synchronize_alerts.md  (typos) (#738)
SEKOIA-IO/documentation: docs/xdr/usecases/playbook/synchronize_alerts.md

## Your Playbook is ready

- **Save** your Playbook by clicking on the `SAVE` button on the center of the page
- **Activate** your Playbook by toggling the value from `Off` to `On`, on the top right of the page

## See your Playbook runs

- On the top right of the page, **click** on `Runs`
- **Confirm** the displayed `Status` is `Succeeded`

> - If you click on a run from the history, you will be redirected to the Graph page. Then you will be able to see the results of each block by clicking on it, as of the following picture:

Reviewer: ```suggestion
- If you click on a run from the history, you will be redirected to the Graph page. There you will be able to see the results of each block by clicking on it, like in the following screen: 

```
